distributed/deploy/spec.py

```diff
@@ -535,7 +545,7 @@ def adapt(
         maximum_cores: int = None,
         minimum_memory: str = None,
         maximum_memory: str = None,
-        **kwargs
+        **kwargs,
```

Comment:
Interestingly I had to change this to make black happy (I tend to try to avoid unrelated changes like this in general) ... I would have thought all the files were black-compliant but maybe black changed its mind about this at one point.

Reply:
Black adds trailing commas if the code is identified to be Python 3.6+ (it isn't legal in Python 3.5) so maybe you added some other feature that changed how black perceived the code?  Regardless, this is fine.